updating notice to have info on breaking changes (#889)
ethereum-optimism/docs: pages/builders/notices/granite-changes.mdx

  *   **Option 2:** Alternatively, chain operators can use the override flags to configure your sequencer node by specifying a time in the future when Granite will activate.
      *   Set  `override.granite`  in both `op-node` and `op-geth` to the UNIX timestamp of the block you want to activate the Granite hardfork or corresponding env vars for this.
      *   In general, run `op-node --help` or `op-geth --help` to see flags, their descriptions and environment variables.

  ### Rollup Configuration Breaking Changes 
  

>   Alt-DA mode has had an name change. What was formerly known as Plasma Mode has been renamed and you may need to update your rollup configuration file. You can find the details on how to do that [here](/builders/chain-operators/features/alt-da-mode#breaking-changes-renaming-plasma-mode-to-alt-da-mode)

Reviewer: ```suggestion
  Alt-DA mode has had a name change. What was formerly known as Plasma Mode has been renamed and you may need to [update your rollup configuration file](/builders/chain-operators/features/alt-da-mode#breaking-changes-renaming-plasma-mode-to-alt-da-mode).
```
